nss-evening-cohort-26/fanfic-fe
# Added search bar

## Description
- Created API call to filter posts by title and content
- Created a search bar component
- Created dynamic page to display search results

## Related Issue
#2

## Motivation and Context
This change is required because the user needs to be able to search posts.

## How Can This Be Tested?
Search for post title or post content and click the submit button or hit enter. Search results should be displayed.

## Types of changes
